src/java.base/share/classes/javax/crypto/spec/OAEPParameterSpec.java

```diff
@@ -89,39 +82,49 @@
+    private final String mgfName;
+    private final AlgorithmParameterSpec mgfSpec;
+    private final PSource pSrc;
 
     /**
-     * Constructs a parameter set for OAEP padding as defined in
-     * the PKCS #1 standard using the default values.
+     * The OAEP parameter set with all default values, i.e. "SHA-1" as message
+     * digest algorithm, "MGF1" as mask generation function (mgf) algorithm,
+     * {@code MGF1ParameterSpec.SHA1} as parameters for the mask generation
+     * function, and {@code PSource.PSpecified.DEFAULT} as the source of the
+     * encoding input.
+     *
+     * @deprecated This field uses the default values defined in the PKCS #1
+     *         standard. Some of these defaults are no longer recommended due
+     *         to advances in cryptanalysis -- see the
+     *         <a href="https://www.rfc-editor.org/rfc/rfc8017#appendix-B.1">Appendix B.1 of PKCS #1</a>
+     *         for more details. Thus, it is recommended to create
+     *         a new {@code OAEPParameterSpec} with the desired parameter values
+     *         using the
+     *         {@link #OAEPParameterSpec(String, String, AlgorithmParameterSpec, PSource)} constructor.
+     *
      */
-    private OAEPParameterSpec() {
-    }
+    @Deprecated(since="19")
+    public static final OAEPParameterSpec DEFAULT = new OAEPParameterSpec(
+            "SHA-1", "MGF1", MGF1ParameterSpec.SHA1,
+            PSource.PSpecified.DEFAULT);
 
     /**
      * Constructs a parameter set for OAEP padding as defined in
      * the PKCS #1 standard using the specified message digest
-     * algorithm <code>mdName</code>, mask generation function
-     * algorithm <code>mgfName</code>, parameters for the mask
-     * generation function <code>mgfSpec</code>, and source of
-     * the encoding input P <code>pSrc</code>.
+     * algorithm {@code mdName}, mask generation function
+     * algorithm {@code mgfName}, parameters for the mask
+     * generation function {@code mgfSpec}, and source of
+     * the encoding input P {@code pSrc}.
      *
-     * @param mdName the algorithm name for the message digest.
-     * @param mgfName the algorithm name for the mask generation
-     * function.
-     * @param mgfSpec the parameters for the mask generation function.
-     * If null is specified, null will be returned by getMGFParameters().
-     * @param pSrc the source of the encoding input P.
-     * @exception NullPointerException if <code>mdName</code>,
-     * <code>mgfName</code>, or <code>pSrc</code> is null.
+     * @param mdName the algorithm name for the message digest
+     * @param mgfName the algorithm name for the mask generation function
+     * @param mgfSpec the parameters for the mask generation function;
+     * if {@code null} is specified, {@code null} will be returned by
+     * {@link #getMGFParameters()}
+     * @param pSrc the source of the encoding input P
+     * @exception NullPointerException if {@code mdName},
```

Comment:
Change `@exception` to the more accepted `@throws` tag.

Reply:
Yes, will change.